src/core/background/object/controller.js

```diff
@@ -208,6 +212,8 @@ define((require) => {
 
 			if (!isSongChanged && !this.isReplayingSong) {
 				this.processCurrentState(newState);
+			} else if (!this.shouldScrobblePodcasts && newState.isPodcast) {
+				this.skipCurrentSong();
```

Comment:
I was thinking about ignore podcasts completely, like YouTube connector ignores videos from non-music categories.

Skip is probably ok, but songs should not be skipped in this place, as `skipCurrentSong` will be called on every state change.

Reply:
> I was thinking about ignore podcasts completely, like YouTube connector ignores videos from non-music categories.

Yeah, it occurred to me that these are more or less instances of the same thing, but for now I resisted the urge to generalize the YouTube option.

Reply:
This works without even adding a new flag, as `isPodcast` is already part of the song info. No visible issues, and the `newState`-related error is gone.